### PR TITLE
Request requirements/information doc, plus spelling/proofreading.

### PR DESCRIPTION
**Pull request by linguinee**
Made a new doc for request information guidelines.

### CODE REVIEW

File: requests.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+To ensure that DxE Tech is able to swiftly help you with your tech request, we need the following information:
+
+* Summary of the problem or request
+  * For tech problems, please include any error messages and screenshots of the issue(s), how long the problem has been happening or when it was discovered, and whom the issue impacts
+  * For tech requests, please include whom the request will impact, how it affects DxE, why it would be nice to have, and how likely it will be used
+* How urgent it is
+  * Is there a hard deadline? Is something else depending on it?
+
+In order to prioritize things appropriately, especially for requests, we need insight into the context behind it. Since we're a volunteer team, we want to dedicate our time to things that will help DxE and its members the most.
```
Comment (mjperrone) [inline requests.md]: + something about not wanting to spend time on things that won't be used?

File: requests.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#Tech Requests
+
+To ensure that DxE Tech is able to swiftly help you with your tech request, we need the following information:
+
+* Summary of the problem or request
+  * For tech problems, please include any error messages and screenshots of the issue(s), how long the problem has been happening or when it was discovered, and whom the issue impacts
```
Comment (samertm) [inline requests.md]: nit: maybe bold "tech problems" and "tech requests" to make the structure of this section more obvious?

File: requests.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  * For tech requests, please include whom the request will impact, how it affects DxE, why it would be nice to have, and how likely it will be used
+* How urgent it is
+  * Is there a hard deadline? Does something else depend on it?
+
+In order to prioritize things appropriately, especially for requests, we need insight into the context behind it. Since we're a volunteer team, we want to dedicate our time to things that will help DxE and its members the most. Requests with the most impact and likelihood to be used will be prioritized.
+
+##Examples
+
+###Problem
```
Comment (samertm) [inline requests.md]: nit: s/Problem/Tech Problem/ ?

File: requests.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+The email being sent to people who sign up for our mailing list recently broke – the header image is no longer showing up. It was working last week. Attached is a screenshot of the broken email, along with the image that's supposed to be showing up. Could someone please fix this asap?
+
+###Request
```
Comment (samertm) [inline requests.md]: nit: s/Request/Tech Request/ ?

**Comment by samertm** [comment]
Looks really good! Only had some small nits, the content all looks good to me tho. :shipit: :ship:

**Comment by samertm** [comment]
lgtm

**Pull request merged**